src/test/ui/useless-doc-comment/trait-impl.rs

```diff
@@ -0,0 +1,14 @@
+//! issue #56766
+//!
+//! invalid doc comment at the end of a trait declaration
+
+trait Foo {
+    ///
+    fn foo(&self);
+    /// I am not documenting anything
+    //~^ ERROR: found a documentation comment that doesn't document anything [E0585]
+}
+
+fn main() {
+
```

Comment:
Add a `let _: usize ="";` line to make sure the compiler recovers from the incorrect doc comment.